contrib/kind.sh

```diff
@@ -216,40 +180,71 @@ ovn_apiServerAddress=${API_IP} \
 kind create cluster --name ${KIND_CLUSTER_NAME} --kubeconfig ${HOME}/admin.conf --image kindest/node:${K8S_VERSION} --config=${KIND_CONFIG_LCL}
 export KUBECONFIG=${HOME}/admin.conf
 cat ${KUBECONFIG}
-mkdir -p /tmp/kind
-sudo chmod 777 /tmp/kind
-count=0
-until kubectl get secrets -o jsonpath='{.items[].data.ca\.crt}'
-do
-  if [ $count -gt 10 ]; then
-    echo "Failed to get k8s crt/token"
-    exit 1
-  fi
-  count=$((count+1))
-  echo "secrets not available on attempt $count"
-  sleep 5
-done
-kubectl get secrets -o jsonpath='{.items[].data.ca\.crt}' > /tmp/kind/ca.crt
-kubectl get secrets -o jsonpath='{.items[].data.token}' > /tmp/kind/token
+
+if [[ "$IP_FAMILY" == "ipv6" ]]; then
+  # IPv6 clusters need some CoreDNS changes in order to work in k8s CI:
+  # 1. k8s CI doesn´t offer IPv6 connectivity, so CoreDNS should be configured
```

Comment:
This isn't "k8s CI" though. Does github CI offer IPv6 connectivity?

Reply:
since there is no IPv6 from the VM to internet, we are in an offline environment and CoreDNS can not reach upstream DNS servers. 
There is (or was, I didn't follow up) a limitation with the CoreDNS default config in Kubernetes with this, so you need to update the CoreDNS configuration to work in an offline environment
xref https://github.com/coredns/coredns/issues/2494#issuecomment-457215452

Reply:
ok, well, so the comment should say "GitHub CI" rather than "k8s CI" then but it sounds like it's otherwise correct

Reply:
can you try removing these lines @Billy99 ?
```
  # 2. k8s CI adds following domains to resolv.conf search field:
  # c.k8s-prow-builds.internal google.internal.
```
and
```
      -e 's/^.*kubernetes cluster\.local/& internal/' \
```

Reply:
@aojea Yes. I'm working through another issue. Then I'll give it try. Tonight or first thing in the morning.

Reply:
Done. Passes CI with lines removed. Also, comment updated to "GitHub CI" rather than "k8s CI"